final release/format_data.py

```diff
@@ -1,200 +1,266 @@
-                vertical_flip = flip,
-                fill_mode = "nearest",
+                rotation_range=rotation_range,
+                zoom_range=zoom_range,
+                width_shift_range=shift_range,
+                height_shift_range=shift_range,
+                horizontal_flip=flip,
+                vertical_flip=flip,
+                fill_mode ="nearest",
                 preprocessing_function=add_noise)
     
     return datagen
 
 
-def data_pregenerate(images, labels, datagen, batch_size, data_mult, seed):
+def data_pregenerate(images, labels, datagen, batch_size, nb_epoch, seed):
     samples = images.shape[0]
+    
+    """Pre-generates data for Sklearn models.
+    
+    Using base set of images, a random combination is augmentations within
+    a defined range is applied to generate a new batch of data.
+    
+    
+    # Arguments
+        images: Array of images with shape (samples, dim, dim, 1)
+        labels: Array of labels
+        datagen: The data generator outputed by the data_gen function.
+        batch_size: Number of sample per batch.
+        nb_epoch: The number of epochs to generate for.
+        seed: Random seed. All our tests used 0.
+        
+    # Returns
+        Array of augmented images and their corresponding labels.
+
+    """
 
     # the .flow() command below generates batches of randomly transformed images
-    # and saves the results to the `preview/` directory
+ 
     gen = datagen.flow(images, labels, batch_size=batch_size, seed=seed)
 
-    pro_images = np.zeros((images.shape[0]*data_mult, images.shape[1], images.shape[2], 1))
-    pro_labels = np.zeros((labels.shape[0]*data_mult, 2))
+    pro_images = np.zeros((images.shape[0] * nb_epoch, images.shape[1],
+                           images.shape[2], 1))
+    pro_labels = np.zeros((labels.shape[0] * nb_epoch, 2))
     
-    for e in range(1, data_mult+1):
+    for epoch in range(1, nb_epoch+1):
         batch = 1
         b = batch_size
-        b_start = samples*(e-1)
+        b_start = samples * (epoch-1)
 
         for X_batch, Y_batch in gen:
-            if batch < (samples/b):
-                pro_images[b_start+b*(batch-1):b_start+batch*b, :, :, :] = X_batch
-                pro_labels[b_start+b*(batch-1):b_start+batch*b, :] = Y_batch
+            if batch < (samples / b):
+                pro_images[b_start + b*(batch-1):b_start + batch*b, :, :, :] = X_batch
+                pro_labels[b_start + b*(batch-1):b_start + batch*b, :] = Y_batch
                 
-            elif batch == (samples/b):
+            elif batch == int(samples / b):
                 break
 
             else: 
-                pro_images[b_start+b*(batch-1):b_start+b*(batch-1) + X_batch.shape[0]%b, :, :, :] = X_batch
-                pro_labels[b_start+b*(batch-1):b_start+b*(batch-1) + X_batch.shape[0]%b, :] = Y_batch
+                pro_images[b_start + b*(batch-1):b_start + b*(batch-1) + X_batch.shape[0]%b, :, :, :] = X_batch
```

Comment:
Spaces around %, maybe add some newlines to make this line easier to read

Reply:
done